openwebif/api.py

```diff
@@ -156,6 +160,19 @@ def default_all(self):
         self.status_info = {}
         self.is_recording_playback = False
 
+    def send_message(self, text: str):
+        """
+        Sends a message to the TV screen.
+
+        :param text: The message to display
+        :return: True if successful, false if there was a problem
+        """
+
+        url = self._base + URL_MESSAGE.format(str(self.message_timeout), text)
```

Comment:
The docs say 

> timeout - (optional, number) seconds to display message; defaults to -1: infinitly


The problem here is `message_timeout` may have a value of `None` which you wont want in the URL. Perhaps check it is not none before adding it to the URL.